bin/crew

```diff
@@ -52,6 +57,8 @@ require_relative '../lib/package'
 require_relative '../lib/package_utils'
 require_relative '../lib/util'
 
+require_gem('highline/import')
```

Comment:
Would it make sense to group all the require* lines together above everything at the top?  It looks a bit disorganized when spread out like this.

Reply:
No. I want the debug require pry-byrbug to come as early as possible, right after const.rb is loaded.